addons/GearSwap/targets.lua

```diff
@@ -28,30 +28,34 @@
 -- Target Processing --
 
 function valid_target(targ)
+    local targ = targ or false
```

Comment:
Why `false`, and why `local`? If it should have a default value just `targ = targ or false` will already be local, since `targ` is a parameter. This way you're introduced a new variable which shadows the parameter.

I think this line can just be removed, no? `false` doesn't get us anything here, unless I'm missing something.